### ISSUE TITLE
Running osx.sh left mac in an unusable state

### ISSUE THREAD

**Issue by sirech** [donnemartin/dev-setup#79]
I ran the osx.h script on a freshly installed mac, and it instantly became unresponsive. Restarting didn't help, as it never got past the login screen (filevault on). I tried again commenting out the SSD parts and the screen parts, and it happened again.

I could not debug it further than this, as I had to basically reinstall OSX again

**Comment by donnemartin** [comment]
@sirech thanks for reporting, is anyone else seeing this?

**Comment by TheDom1** [comment]
I had it happen the first time I installed as well. Had to wipe my Mac and start over.

**Comment by mosn** [comment]
Yup me too. wish I've been here first. The problem seems to be with root permission. My disk was showing a user with FFFFEEEE-DDDD 

Couple of restarts and suddenly kernel panic.
